sdk/cosmos/azure-cosmos/tests/test_headers_async.py

```diff
@@ -236,5 +247,24 @@ async def test_partition_merge_support_header(self):
         # base method to set the header(GetHeaders).
         await self.container.read(raw_response_hook=partition_merge_support_response_hook)
 
+    async def test_client_level_priority_level_async(self):
+        # Test that priority level set at client level is used for all requests
+        CosmosClient(self.host, self.masterKey,
+            priority_level=client_priority_level,
+            raw_response_hook=client_priority_raw_response_hook)
```

Comment:
this won't initialize the async client and run any requests - you will need to use either an async context manager or await the `__aenter__()` function to make the client do its warmup calls